workspaces/editors.md

```diff
@@ -178,3 +179,36 @@ participants. If they do, click **Accept** to proceed.
 
 At this point, you'll be able to share your project and work with your partner
 in real-time.
+
+## RStudio
+
+Coder supports [RStudio](rstudio.com). To create a workspace that lets you use
+RStudio:
+
+1. Create an image that includes RStudio (see
+   [rocker/rstudio](https://hub.docker.com/r/rocker/rstudio) for an example).
+
+1. [Create a workspace](getting-started.md#2-create-a-workspace) using the image
+   you created in the previous step.
+
+1. Because RStudio uses an authentication model that matches to your Linux user
+   with a home directory, whereas your image likely uses the `coder` user, we
+   recommend making a new user for your RStudio projects. To do so, go to
+   **Applications** > **Terminal** in Coder and run:
```

Comment:
I'd adjust the wording here to be more around user with home directory and known password. Something along these lines:

1. (Optional) RStudio requires authentication using a Linux user and stores all files in their home directory. If from your Docker image you're using the root user, `coder` user, or a user that either does not have a home directory or known password, then we recommend creating a new user for your RStudio projects. To do so, go to...

```console
useradd --create-home <username>
passwd <username>
```